### PR TITLE
Publishy

### PR DESCRIPTION
**Pull request by DFurnes**
# Changes
Removing the automatic Wercker `npm publish` step for now since it seemed like an awkward flow. Also adds `grunt build` to the `prepublish` script so you can't publish dev builds.

For review: @DoSomething/front-end

### CODE REVIEW

File: wercker.yml [hunk 1]
```diff
@@ -8,19 +8,10 @@ build:
         tasks: build
     - npm-test
 
-# Automatically deploy `master` branch to Github pages & publish
-# package to NPM if version is not published to registry.
+# Automatically deploy `dev` branch to Github pages
```
Comment (sbsmith86) [inline wercker.yml]: :-1:

Comment (sbsmith86) [inline wercker.yml]: :+1:

**Comment by sbsmith86** [comment]
So we don't have to merge into master to deploy? :clap:

**Comment by DFurnes** [comment]
Yup yup. :fire:

**Pull request merged**